packages/boot/test/bootstrapTests/liquidation.ts

```diff
@@ -205,16 +242,133 @@ export const makeLiquidationTestContext = async t => {
     },
   };
 
+  const setupVaults = async (
+    collateralBrandKey: string,
+    managerIndex: number,
+    setup: LiquidationSetup,
+  ) => {
+    await setupStartingState({
+      collateralBrandKey,
+      managerIndex,
+      price: setup.price.starting,
+    });
+
+    const minter =
+      await walletFactoryDriver.provideSmartWallet('agoric1minter');
+
+    for (let i = 0; i < setup.vaults.length; i += 1) {
+      const offerId = `open-${collateralBrandKey}-vault${i}`;
+      await minter.executeOfferMaker(Offers.vaults.OpenVault, {
+        offerId,
+        collateralBrandKey,
+        wantMinted: setup.vaults[i].ist,
+        giveCollateral: setup.vaults[i].atom,
+      });
+      t.like(minter.getLatestUpdateRecord(), {
+        updated: 'offerStatus',
+        status: { id: offerId, numWantsSatisfied: 1 },
+      });
+    }
+
+    // Verify starting balances
+    for (let i = 0; i < setup.vaults.length; i += 1) {
+      check.vaultNotification(managerIndex, i, {
+        debtSnapshot: {
+          debt: { value: scale6(setup.vaults[i].debt) },
+        },
+        locked: { value: scale6(setup.vaults[i].atom) },
+        vaultState: 'active',
+      });
+    }
+  };
+
+  const placeBid = async (
```

Comment:
this is placing multiple bids
```suggestion
  const placeBids = async (
```